.rubocop.yml

```diff
@@ -0,0 +1,153 @@
+Metrics/LineLength:
+  Enabled: false
+  Exclude:
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+  Max: 100
+
+# [codesmell]
+# I don't really get the point of this cop.
+Performance/RedundantMerge:
+  Enabled: false
+
+# Do not use "and" or "or" in conditionals, but for readability we can use it
+# to chain executions. Just beware of operator order.
+Style/AndOr:
+  EnforcedStyle: conditionals
+
+# braces_for_chaining seems a good fit of what we've been doing so far.
+Style/BlockDelimiters:
+  EnforcedStyle: braces_for_chaining
+  IgnoredMethods:
+    - expect
+
+# Double empty lines are useful to separate conceptually different methods
+# in the same class or module.
+Style/EmptyLines:
+  Enabled: false
+
+Style/EmptyLinesAroundBlockBody:
+  Exclude:
+    # RSpec is all made of blocks. Disable this config in RSpec
+    # to be consistent with EmptyLinesAroundClassBody and EmptyLinesAroundModuleBody
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+
+# In most cases, a space is nice. Sometimes, it's not.
+# Just be consistent with the rest of the surrounding code.
+Style/EmptyLinesAroundClassBody:
+  Enabled: false
+
+# In most cases, a space is nice. Sometimes, it's not.
+# Just be consistent with the rest of the surrounding code.
+Style/EmptyLinesAroundModuleBody:
+  Enabled: false
+
+# I personally don't care about the format style.
+# In most cases I like to use %, but not at the point I want to enforce it
+# as a convention in the entire code.
+Style/FormatString:
+  Enabled: false
+
+# Prefer the latest Hash syntax
+Style/HashSyntax:
+  Exclude:
+    # But Rakefiles generally have some definition like
+    #   :default => :test
+    # that looks nicer with the old rocket syntax.
+    - 'Rakefile'
+
+# Array indentation should be considered like MultilineMethodCallIndentation indentation
+# and use 4 spaces instead of 2.
+Style/IndentArray:
+  IndentationWidth: 4
+
+# Hash indentation should be considered like MultilineMethodCallIndentation indentation
+# and use 4 spaces instead of 2.
+Style/IndentHash:
+  IndentationWidth: 4
+
+# [codesmell]
+# It's not always that bad.
+Style/IfInsideElse:
+  Enabled: false
+
+# Multi-line differs from standard indentation, they are indented twice.
+Style/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+  IndentationWidth: 4
+
+# unless is not always cool.
+Style/NegatedIf:
+  Enabled: false
+
+# Magic numbers are not welcomed
+Style/NumericLiterals:
+  Exclude:
+    # however tests can use numeric literals for method calls,
+    # without the need to define a variable just for that.
+    - 'spec/**/*_spec.rb'
+    - 'test/**/*_test.rb'
+
+# There are cases were the inline rescue is ok. We can either downgrade the severity,
+# or rely on the developer judgement on a case-by-case basis.
+Style/RescueModifier:
+  Enabled: false
+
+# It's nice to be consistent. The trailing comma also allows easy reordering,
+# and doesn't cause a diff in Git when you add a line to the bottom.
```

Comment:
Was wondering were this was coming from when I saw the diff. It sounds like a good idea (not sure why I haven't done it before).